### ISSUE TITLE
Automatically detect changes in dependencies

### ISSUE THREAD

**Issue by fedj** [openzipkin/zipkin#1255]
The dependency graph is a really nice feature that allows us to have an always up to date documentation of the architecture of the platform. Thus, we tend to look at it once in a while and don't really pay attention to new things.
The idea here would be to emphasize new dependencies or new services in the graph.
Two solutions come to mind:
1. Highlight "new things" in the graph
   - Pros: Quick to develop, nice to look at and fairly intuitive
   - Cons: We would have to define what's "new" as it could fade away quickly. What about removed services?
2. Send a notification on changes
   - Pros: Changes in the graph could be seen as events and a notification seems appropriate
   - Cons: We could quickly get tired of notifications (if you activate it on preprod or if you build new services quickly)

What do you think ?

**Comment by eirslett** [comment]
Sounds like a useful feature! Maybe it could be a separate application that uses zipkin's API?

**Comment by fedj** [comment]
Glad to see I'm not the only one interested in this one !
Defining it as a separate application would go in the way of the second solution with notifications.
It could be either embedded in the dependency job or a completely new app triggered by the end of the dependency job.

**Comment by jorgheymans** [comment]
I agree with @eirslett that sites using the trace data to manage their architecture dependencies should build such capability themselves outside of Zipkin. 

@fedj are you ok with closing this ?
